resources/views/site/blocks/feature_block.blade.php

```diff
@@ -125,8 +125,8 @@
                             @elseif( $feature_type === 'videos')
                                 <em class="type f-tag">Video</em>
                             @endif
-                            @if ($item->title)
-                                <strong class="title f-list-3">{!! $item->present()->title !!}</strong>
+                            @if ($item->title_display || $item->title)
+                                <strong class="title f-list-3">{!! $item->present()->title_display ?? $item->present()->title !!}</strong>
                             @endif
```

Comment:
I don't know that the `@if` condition adds anything here, since we're falling back to the title whether or not it has a value (which is always should). I think you can simplify this by removing the `@if` and just outputting the `<strong>` with the `??` values.